ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -361,7 +390,7 @@ else
 	if [ $USE_IO == yes -a $IO_PRESENT == no ]
 	then
 		echo "PX4IO not found" >> $LOG_FILE
-		tune_control play -m ${TUNE_ERR}
+		tune_control play -t 2
```

Comment:
Out of scope, just an idea: a label would be nice here, for example `tune_control play -t error`

Reply:
Hi @bkueng , we can do that, I can put TUNE_ERR back in and give it a value of 1.